src/include/meta.h

```diff
@@ -19,6 +19,7 @@
  */
 #define WT_BACKUP_TMP "WiredTiger.backup.tmp"  /* Backup tmp file */
 #define WT_METADATA_BACKUP "WiredTiger.backup" /* Hot backup file */
+#define WT_EXPORT_BACKUP "WiredTiger.export"   /* Export backup file */
```

Comment:
Alphabetize.

Reply:
(Not sure where to put this as it is an omission so I'm putting it here.)
You should have cleanup code in case the system crashes while that cursor is open and the file exists. I think the code at the beginning of `meta/meta_turtle.c:wt_turtle_init` is the place you should look as we do other metadata related cleanup there.

Reply:
@sueloverso Just to clarify, did you mean that we should remove the `WiredTiger.export` file if it exists before attempting to open the `backup:export `cursor?

Reply:
Thanks for pushing back because on further though we *cannot* remove it on restart, otherwise that would not work for a destination database. Yes, I think it should be removed if it exists before opening the export cursor. But that should happen after the check where we cannot have any other backup cursor open, so probably as late was the time when we're going to open the `WiredTiger.export` file.

Reply:
Make sure to add a test case for that situation. The python test might need to look something like this:
Create tables 1 and 2.
Open export cursor on main database.
Copy main database to another directory, including the WiredTiger.export
Open a connection and session on the directory copy
Create table 3, drop table 2
Open the export cursor on the database copy
Make sure table 3 exists in that WiredTiger.export
Make sure table 2 does not exist in WiredTiger.export
Close export cursor

Reply:
I temporarily changed the `copy_wiredtiger_home` method to also copy directories and I can verify that the information for table 3 exists in the `WiredTiger.export` file while the information for table 2 does not exist.

Reply:
@mm-ng can you try this fix. I have it locally, but I think it suits your PR more than mine:

--- a/test/suite/helper_tiered.py
+++ b/test/suite/helper_tiered.py
@@ -124,8 +124,10 @@ class TieredConfigMixin:
             return ''
 
         # Setup directories structure for local tiered storage.
-        if self.is_local_storage and not os.path.exists(self.bucket):
-            os.mkdir(self.bucket)
+        if self.is_local_storage:
+            bucket_full_path = os.path.join(self.home, self.bucket)
+            if not os.path.exists(bucket_full_path):
+                os.mkdir(bucket_full_path)
 
         # Build tiered storage connection string.
         return \

Reply:
Sorry, don't understand what's happening with the formatting in my comment above, can't make it look right...

Reply:
I gave that a try but seems like the bucket directory is still not being created in the new directory.

Reply:
Yes, I can confirm that the test does work as expected with normal non-tiered storage.

Reply:
Thanks for the test. Please alphabetize (that tiny comment at the top got lost in the long thread of the testing, my fault for putting that discussion here).